src/components/filter/filter-comp/categoryFilter-comp/category-filter.js

```diff
@@ -0,0 +1,35 @@
+export function CategoryFilter() {
+    return (
+        <div className="category-filter">
+            <span className="sort-heading">Sort by category</span> <br />
+            <span className="sort-heading">Men</span>
+            <div className="men-categories">
+                <div>
+                    <input type="checkbox" id="tshirt"></input>
+                    <label htmlFor="tshirt">Tshirt</label>
+                </div>
+                <div>
+                    <input type="checkbox" id="jeans"></input>
+                    <label htmlFor="jeans">Jeans</label>
+                </div>
+            </div>
+            <div></div>
+            <span className="sort-heading">Women</span>
+            <div className="women-categories">
+                <div>
+                    <input type="checkbox" id="dresses"></input>
+                    <label htmlFor="dresses">Dresses</label>
+                </div>
```

Comment:
Can put catrgories in array and then render it in UI,instead of doing statically.so, that if categories change in future.Dont have to change everything manually